### ISSUE TITLE
No Syntax Highlighting in `nvim_treesitter`

### ISSUE THREAD

**Issue by LordMZTE** [vantreeseba/tree-sitter-haxe#1]
Hello, I installed this parser using this config in my `init.lua` file:
```lua
local configs = require "nvim-treesitter.configs"
local parsers = require "nvim-treesitter.parsers"

local parser_config = parsers.get_parser_configs()

parser_config.haxe = {
    install_info = {
        url = "https://github.com/vantreeseba/tree-sitter-haxe",
        files = {"src/parser.c"},
        branch = "main"
    },
    filetype = "haxe"
}

configs.setup {
    ensure_installed = { "haxe" },
    highlight = {
        enable = true
    }
}

```

and it installs just fine when running `:TSInstallFromGrammar haxe`. However, I don't get any syntax highlighted in .hx files (Their filetype is set to `haxe`), and running `:checkhealth nvim_treesitter` gives me this output:
```
health#nvim_treesitter#check
========================================================================
## Installation
  - OK: `tree-sitter` found  0.19.4 (parser generator, only needed for :TSInstallFromGrammar)
  - OK: `node` found v15.6.0 (only needed for :TSInstallFromGrammar)
  - OK: `git` executable found.
  - OK: `cc` executable found. Selected from { vim.NIL, "cc", "gcc", "clang", "cl" }
  - OK: Neovim was compiled with tree-sitter runtime ABI version 13 (required >=13). Parsers must be compatible with runtime ABI.

## Parser/Features H L F I
  - haxe           . . . . 

 Legend: H[ighlight], L[ocals], F[olds], I[ndents]
         +) multiple parsers found, only one will be used
         x) errors found in the query, try to run :TSUpdate {lang}
```

and that looks fine to me. However the whole haxe file has no syntax highlighting.

**Comment by LordMZTE** [comment]
Having installed some more treesitter parsers, it looks like all others show a checkmark instead of `.` in the syntax column. So apparently, highlighting is disabled for haxe. Is this something that needs to be changed here, or is that something I can change using my config?

**Comment by vantreeseba** [comment]
I'm not sure how they are doing it with other parsers, but I had to manually copy the queries/*.scm into the tree-sitter plugin folder.

I will look into this, and see how other ones are doing it, but it looks like they just include it in the main repo.

For example: https://github.com/nvim-treesitter/nvim-treesitter/tree/master/queries/javascript

So in my local install of it, I just made a folder beside the JS one called haxe, copied my SCM into it, restarted vim and that worked.
